integrations/php/workload-executor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use MongoDB\Tests\UnifiedSpecTests\EntityMap;
+use MongoDB\Tests\UnifiedSpecTests\Loop;
+use MongoDB\Tests\UnifiedSpecTests\UnifiedTestCase;
+use MongoDB\Tests\UnifiedSpecTests\UnifiedTestRunner;
+use function MongoDB\BSON\fromJSON;
+use function MongoDB\BSON\toPHP;
+
+require_once __DIR__ . '/vendor/autoload.php';
+require_once __DIR__ . '/vendor/bin/.phpunit/phpunit/vendor/autoload.php';
+
+$uri = $argv[1];
+$json = toPHP(fromJSON($argv[2]));
+
+$entityMapObserver = function (EntityMap $entityMap) {
+    $numIterations = $entityMap->offsetExists('iterations') ? $entityMap['iterations'] : -1;
+    $numSuccesses = $entityMap->offsetExists('successes') ? $entityMap['successes'] : -1;
+    $errors = $entityMap->offsetExists('errors') ? $entityMap['errors'] : [];
+    $failures = $entityMap->offsetExists('failures') ? $entityMap['failures'] : [];
+    $events = $entityMap->offsetExists('events') ? $entityMap['events'] : [];
+
+    $numErrors = $entityMap->offsetExists('errors') ? count($errors) : -1;
+    $numFailures = $entityMap->offsetExists('failures') ? count($failures) : -1;
+
+    file_put_contents('events.json', json_encode(compact('events', 'errors', 'failures')));
+    file_put_contents('results.json', json_encode(compact('numErrors', 'numFailures', 'numSuccesses', 'numIterations')));
+};
+
+$runner = new UnifiedTestRunner($uri);
+$runner->allowKillAllSessions(false);
+$runner->setEntityMapObserver($entityMapObserver);
+
+Loop::setSleepUsecBetweenIterations(10000);
+
+pcntl_async_signals(true);
+pcntl_signal(SIGINT, function() {
+    Loop::allowIteration(false);
+});
+
+$tests = UnifiedTestCase::fromJSON($json);
+
+foreach ($tests as $test) {
+    $runner->run($test);
```

Comment:
Note: I believe this needs to catch exceptions and count them as either errors/failures. See [Behavior Description: #4](https://mongodb-labs.github.io/drivers-atlas-testing/spec-workload-executor.html#behavioral-description) from the workload executor docs.